project/AppDependencies.scala

```diff
@@ -4,28 +4,30 @@ import sbt._
 object AppDependencies {
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "govuk-template" % "5.56.0-play-26",
-    "uk.gov.hmrc" %% "play-ui" % "9.0.0-play-26",
-    "uk.gov.hmrc" %% "bootstrap-play-26" % "1.9.0",
+    "uk.gov.hmrc" %% "govuk-template" % "5.69.0-play-28",
+    "uk.gov.hmrc" %% "play-ui" % "9.6.0-play-28",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.3.0",
     "com.beachape" %% "enumeratum" % "1.6.1",
-    "uk.gov.hmrc" %% "simple-reactivemongo" % "7.30.0-play-26",
+    "uk.gov.hmrc" %% "time" % "3.25.0",
+    "uk.gov.hmrc" %% "simple-reactivemongo" % "8.0.0-play-28",
 
-    "uk.gov.hmrc" %% "time-to-pay-taxpayer-cor" % "[0.34.0]",
+    "uk.gov.hmrc" %% "time-to-pay-taxpayer-cor" % "0.40.0",
 
-    "uk.gov.hmrc" %% "domain" %  "5.9.0-play-26",
-    "com.typesafe.play" %% "play-json-joda" % "2.6.13",
+    "uk.gov.hmrc" %% "domain" %  "6.0.0-play-28",
+    "com.typesafe.play" %% "play-json-joda" % "2.7.4",
 
-    "com.beachape" %% "enumeratum-play" % "1.5.15", // Nb: Staying below version 1.5.16 whilst https://github.com/lloydmeta/enumeratum/issues/285 causes play-2.7 to be pulled in
-    "org.julienrf" %% "play-json-derived-codecs" % "4.0.1"// Nb: Choose carefully for version of play. 4.0.1 is latest for play 2.6.
+    "com.beachape" %% "enumeratum-play" % "1.6.1", // Nb: Staying below version 1.5.16 whilst https://github.com/lloydmeta/enumeratum/issues/285 causes play-2.7 to be pulled in
```

Comment:
If we are at play 2.8 we can remove these comments https://github.com/lloydmeta/enumeratum/issues/285 causes play-2.7 to be pulled in